en/asgardeo/docs/guides/authentication/mfa/add-duo-login.md

```diff
@@ -0,0 +1,65 @@
+# Add Duo login
+
+[Duo](https://duo.com/) is a multi factor authentication option that offer multiple verification methods such as push notifications, SMS, phone calls, and hardware tokens.
+
+In this guide, you'll learn how to integrate Duo with your Asgardeo application, enabling Duo as a second-factor authentication provider to enhance your application's security.
+
+## Prerequisites
+
+Before you begin, ensure you have the following:
+
+- Duo Account: A Duo administrator account to configure your Duo application.
+- Asgardeo Account: Access to your Asgardeo console with administrative privileges.
+
+## Configure Duo Security Application
+
+1. Log in to the [Duo Securty admin panel](https://admin.duosecurity.com/login). 
+2. Click **Applications** from the left panel and then click the **Protect an Application** button.
+
+    ![Create duo security app]({{base_path}}/assets/img/guides/mfa/duo/add-duo-security-app.png){: style="display: block; margin: 0; border: 0.3px solid lightgrey;"}
+
+3. On the **Protect an Application page**, search for **Web SDK** and click on **Protect**.
+
+    ![Protect web SDK app]({{base_path}}/assets/img/guides/mfa/duo/protect-web-sdk-app.png){: style="display: block; margin: 0; border: 0.3px solid lightgrey;"}
+
+4. Save the **Client ID**, **Client secret**, and **API hostname**, as you will need these later.
+
+## Add Duo as a Connection in Asgardeo
+
```

Comment:
before beginning the steps, add an additional sentence.

Something like "Follow the steps below to...."